sql/src/main/java/io/crate/analyze/symbol/Aggregations.java

```diff
@@ -22,11 +22,18 @@
 
 package io.crate.analyze.symbol;
 
+import io.crate.analyze.validator.LiteralVisitor;
 import io.crate.metadata.FunctionInfo;
+import io.crate.operation.scalar.arithmetic.ArrayFunction;
+
+import java.util.List;
 
 public final class Aggregations {
 
     private final static AggregationSearcher AGGREGATION_SEARCHER = new AggregationSearcher();
+    private final static AggregationOrScalarSearcher AGGREGATION_OR_SCALAR_SEARCHER = new AggregationOrScalarSearcher();
+    private final static GroupBySymbolMatcher GROUP_BY_MATCHER = new GroupBySymbolMatcher();
+    private final static LiteralVisitor LITERAL_VISITOR = new LiteralVisitor();
```

Comment:
Use the new public INSTANCE also in GroupBySymbolValidator and remove declaration from here: https://github.com/crate/crate/pull/5943/files#diff-e2ef8429cbc655c07d660ac6d159557dR38